src/app/payment/paymentMethod/StripePaymentMethod.tsx

```diff
@@ -94,13 +98,22 @@ const StripePaymentMethod: FunctionComponent<StripePaymentMethodProps & WithChec
         return stripeInitializeOptions[paymentMethodType];
     }, [paymentMethodType, getIndividualCardElementOptions]);
 
-    const initializeStripePayment = useCallback(async (options: PaymentInitializeOptions) => {
+    const validateInstrument = (_shouldShowNumber: boolean, selectedInstrument: CardInstrument) => {
```

Comment:
do you need the first param? `_shouldShowNumber` doesn't seem to be used

Reply:
Stripe does not use this parameter but Adyenv2 does, if we  dont inlcude this param we have an error.
We are using `validateInstrument` prop in order to pass this callback to `HostedWidgetPaymentMethod` and reuse this logic.